about.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/3866333966.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <aside class="menu-bar">
+        <i class="fa fa-close fa-2x hamburger white" id="close"></i>
+        <ul class="menu-bar-ul">
+            <li class="mobile-list"><a href="index.html" class="white">Main Page</a></li>
+            <li class="mobile-list"><a href="#" class="white">Program</a></li>
+            <li class="mobile-list"><a href="#" class="white">Join</a></li>
+            <li class="mobile-list"><a href="#" class="white">Sponsor</a></li>
+            <li class="mobile-list"><a href="#" class="white">News</a></li>
+            <li class="mobile-list"><a href="#" class="navbar-cc-link white">CC Campaign</a></li>
+        </ul>
+    </aside>
+    <!--THE NAVIGATION BAR SECTION-->
+    <nav class="nav">
+        <div class="black-navbar">
+            <ul class="black-navbar-ul">
+                <li><i class= "fa fa-facebook white"></i></li>
+                <li><i class="fa fa-twitter white"></i></li>
+                <li class="white">Logout</li>
+                <li class="white">MyPage</li>
+                <li class="white">Korean</li>
+            </ul>
+        </div>
+        <div class="main-navbar">
+            <ul class="navbar-logo">
+                <li><a href="index.html"><i class="fa fa-life-ring fa-3x orange home-page-link"></i></a></li>
+                <li><h4>CC Global <br> Summit 2015</h4></li>
+            </ul>
+            <ul class="main-navbar-ul">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#" class="navbar-cc-link">CC Campaign</a></li>
+            </ul>
+        </div>
+        </nav>
+    <!--THE MAIN PAGE SECTION-->
+    <section class="first-section">
```

Comment:
- [ ] The font style is not correct for this section and for other sections of the about page. Let's try to fix this issue by applying the correct `font family` so that it should match the original design.

Your
![image](https://user-images.githubusercontent.com/53019916/127885791-bc155c9e-ce6c-4236-82dd-f299b44ec807.png)

Original
![image](https://user-images.githubusercontent.com/53019916/127885886-e4345103-93a6-4dce-9520-1e467fb5e6c7.png)